common/SmokeTests/SmokeTest/Update-Dependencies.ps1

```diff
@@ -31,63 +35,79 @@ function Log-Warning($message) {
     }
 }
 
-Register-PackageSource `
-    -Name $NIGHTLY_FEED_NAME `
-    -Location $NIGHTLY_FEED_URL `
-    -ProviderName Nuget `
-    -ErrorAction SilentlyContinue `
+function GetAllPackages {
+    $packages = Find-Package -Source $PACKAGE_FEED_URL -AllVersion -AllowPrereleaseVersions
+    if ($Daily) {
```

Comment:
Shouldn't we do the opposite filtering as well and exclude all the alpha packages if we Daily isn't set?

Reply:
Chatted offline, I need to verify that all of the smoke test dependencies have published GA/beta versions. If so, then we can use that logic.

Reply:
I updated the dependencies to all the latest ga/beta versions, as they all have instances of those published.